src/security/pci-guidance.md

```diff
@@ -0,0 +1,151 @@
+# PCI DSS Responsibility Guidance
+
+Platform.sh has many PCI certified customers using our services. Below we will attempt to clarify the PCI compliance responsibilities for Platform.sh and our customers.
+
+Simple audit questions can be handled via support tickets however in-depth support for your audit may required additional arrangements. Please contact your Platform.sh account manager to discuss your needs.
```

Comment:
Suggested verbage: "Basic PCI questions can be handled by our support team via a ticket.  For more advanced questions or walkthrough of a full audit please contact your Platform.sh Account Manager.